content/index.md

```diff
@@ -55,34 +37,37 @@ develop both our server-side code and our client-side code using components.
 </div>
 </div>
 
-The complate approach is similar to server-side rendering in component-based
-frameworks like [React](https://reactjs.org/) and
-[Vue](https://vuejs.org/), but because their implementations have to deal with
-client-side rendering as well, they are larger and more difficult to port to
-other platforms outside of the JavaScript ecosystem.
-
-This is where complate shines: Because of the
-[lean underlying implementation](https://www.innoq.com/en/blog/self-contained-custom-elements/), it is possible to write a component
-which can easily be reused across multiple projects on different platforms.
-
-Cross-Platform Support
-----------------------
-
-With complate, components can be used across multiple platforms.
-[On the server](https://github.com/complate/complate-stream), we can integrate
-complate macros in different tech stacks like:
-
-* [Express](https://github.com/complate/complate-express) (Node.js)
-* [Ruby on Rails](https://github.com/complate/complate-ruby) (Ruby)
-* [Spring](https://github.com/complate/complate-spring) (Java)
-* etc.
-
-This is particularly useful in heterogeneous software systems, where we may be
-running multiple services using different back-end technologies, but we want to
-reuse front-end components between services. If this is the case, we can
-develop a [pattern library](https://github.com/complate/complate-fractal) using
-complate and then publish our macros for reuse in other services.
-
-We can also use complate
-[on the client](https://github.com/complate/complate-dom) and as a
-[static-site generator](https://github.com/complate/complate-ssg).
+Stateless HTML Rendering
+------------------------
+
+complate focuses on efficiently generating HTML in a fire-and-forget fashion.
+We believe that [server-side rendering should be the default](https://www.innoq.com/de/articles/2020/01/javascript-in-ma%C3%9Fen/), and we created a templating library that is optimized for it.
```

Comment:
IMO complate doesn't require you to buy into INNOQ's belief system, at least not to the full extent: Not everyone's on the frontlines of current/unfortunate trends in web development (lucky them!), so from a less PTSD-laden perspective, complate just offers a nicer* way to render HTML on the server, which you might never have questioned in the first place (bless your heart!).

(Also, that article's in German... 🙂 In my local WIP version, I'll try to work in a reference to [adactio's piece on hydration](https://adactio.com/journal/16404), which seems to nicely encapsulate the various dimensions of this issue. Of course that assumes people actually read and follow links; a fatally flawed premise.)

\* arguably so - that is, _if_ you need the modularity and markup abstractions provided by JSX components or have use for reuse across tech stacks

Reply:
The article is actually in English (https://www.innoq.com/en/articles/2020/01/javascript-in-ma%C3%9Fen/). :face_palm: and I translated it so that I we could link to an article on the benefits of SSR.

Reply:
I'm perfectly OK with making the text less defensive (it's just how it comes out when I write about SSR 🙈). My intent here is to explicitly state our goal with complate: that we want templates that work wonderfully with SSR.

Reply:
I rewrote this section to say that complate is _ideal_ for server-side rendering. We don't want to force someone to buy into a belief system, but I think we should emphasize that it is created with SSR in mind. If someone is already sold on CSR, there are other frameworks they can look into.